### ISSUE TITLE
IME tips not show right placekh

### ISSUE THREAD

**Issue by ltu00** [adirh3/Fluent-Search#110]
**Describe the bug**
IME tips not show right place

**To Reproduce**
1. start search bar ,shift IME to chinese,and type

**Expected behavior**
IME tips not show near search bar

**Screenshots**
![image](https://user-images.githubusercontent.com/50564608/121830730-45352a80-ccf8-11eb-9689-5bbf3384c578.png)

![image](https://user-images.githubusercontent.com/50564608/121830608-f1c2dc80-ccf7-11eb-8cd7-2a9590dad0e5.png)

**Desktop (please complete the following information):**
 - Windows 10 Version: [20H1]
 - Fluent Search Version [0.9.85.50]

**Comment by Metroid-Prime** [comment]
@ltu00  I assigned this low priority looking at your pic.  Let me know if you feel strongly if it qualifies for medium priority.

**Comment by ltu00** [comment]
> @ltu00 I assigned this low priority looking at your pic. Let me know if you feel strongly if it qualifies for medium priority.
when i launched FS on second screen , IME tips will shown on first screen.

**Comment by adirh3** [comment]
@ltu00 this should be fixed on the latest nightly

**Comment by u-Sir** [comment]
> @ltu00 this should be fixed on the latest nightly

it happened again in 0.9.91.14 😂

**Comment by u-Sir** [comment]
> @ltu00 this should be fixed on the latest nightly

![图片](https://user-images.githubusercontent.com/93647031/153702132-f463ceba-f0cd-4ce9-b35f-889898efd208.png)

**Comment by adirh3** [comment]
> > @ltu00 this should be fixed on the latest nightly
> 
> ![图片](https://user-images.githubusercontent.com/93647031/153702132-f463ceba-f0cd-4ce9-b35f-889898efd208.png)

That doesn't happen on stable right? It started to happen only after 0.9.91?

**Comment by u-Sir** [comment]
> 

sorry, I skipped the stable version. Now is 0.9.91.14

**Comment by u-Sir** [comment]
> > > @ltu00 this should be fixed on the latest nightly
> > 
> > 
> > ![图片](https://user-images.githubusercontent.com/93647031/153702132-f463ceba-f0cd-4ce9-b35f-889898efd208.png)
> 
> That doesn't happen on stable right? It started to happen only after 0.9.91?

😂, I skipped the stable version. Now is 0.9.91.14

**Comment by adirh3** [comment]
@u-Sir does it happen on latest nightly?

**Comment by u-Sir** [comment]
> @u-Sir does it happen on latest nightly?

If using `tab` to switch tag, the position still shows over the tag.
![screenshots](https://user-images.githubusercontent.com/93647031/170153158-da04e68b-3bd4-4b6e-8259-5f54b1559ddc.gif)
